src/App.js

```diff
@@ -1,26 +1,10 @@
-import React from 'react';
-import logo from './logo.svg';
+import Calculator from './components/Calculator';
 import './App.css';
 
 function App() {
   return (
```

Comment:
- [ ] The App.js component should be class-based. Let's fix that

Reply:
Hi Code Reviewer, sorry I completed all my project before realizing I was supposed to submit in different part. Please permit this time I and I will be conscious the next time. I have a very short time to implement new changes NOW. kind Regrads